src/helpers/ParseQuery.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Returns what the input is (zip code, county name, or address)
+ * @Param str The string to parse
+ * @Returns A string telling what the input is
+ */
+export default function Parse(str: string): string {
```

Comment:
One thing I forgot to mention earlier was that you could change the return type here from `string` to `'zip' | 'country' | 'address' | 'error'` to make it a bit more explicit. If it looks like a lot to put on that line you could make a new type called `SearchQueryType` or something similar and use it here